02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/Open in Obsidian.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases:
+  -
+tags:
+  -  evergreen
+publish: true
+---
+
+# Open in Obsidian
+
+Official website: https://routinehub.co/shortcut/11156/
```

Comment:
That URL never loads for me, after a couple of minutes of waiting. I'm in the UK. Is an account needed to see it?
(later - it got there in the end. It's just very slow)